var/spack/repos/builtin/packages/kaldi/package.py

```diff
@@ -38,6 +38,8 @@ class Kaldi(Package):    # Does not use Autotools
     url      = "https://github.com/kaldi-asr/kaldi/archive/master.zip"
 
     version('master', git='https://github.com/kaldi-asr/kaldi.git')
+    version('6f2140', git='https://github.com/kaldi-asr/kaldi.git',
+            commit='6f2140b032b0108bc313eefdca65151289642773')
     version('c024e8', git='https://github.com/kaldi-asr/kaldi.git',
```

Comment:
Can you use the commit date, like `2018-07-11`, as the version number instead of the hash? Otherwise order will be meaningless and the default version may not be the newest.

Reply:
I can do, I just used a hash to be inline with what was ther already